### ISSUE TITLE
Yoink+ crash versus Jester

### ISSUE THREAD

**Issue by TerryCavanagh** [TerryCavanagh/diceydungeons.com#543]
```
crashed:    2018-09-30 18:16:35
duration:    00:37:50
error:        Null Object Reference
stack:
elements.Fighter.getequipmentposition (elements/Fighter.hx line 651)
elements.Action.givemultiequipment (elements/Action.hx line 1834)
elements.Action.execute (elements/Action.hx line 1334)
elements.Equipment.doequipmentaction (elements/Equipment.hx line 1775)
Reflect.callMethod (C:\coding\haxe\haxe\std/cpp/_std/Reflect.hx line 55)
motion.actuators.GenericActuator.complete (motion/actuators/GenericActuator.hx line 155)
motion.actuators.SimpleActuator.update (motion/actuators/SimpleActuator.hx line 501)
motion.actuators.SimpleActuator.stage_onEnterFrame (motion/actuators/SimpleActuator.hx line 579)
openfl.events.EventDispatcher.__dispatchEvent (openfl/events/EventDispatcher.hx line 443)
openfl.display.DisplayObject.__dispatch (openfl/display/DisplayObject.hx line 1236)
```

**Comment by TerryCavanagh** [comment]
https://pastebin.com/raw/wGwfiuMg

**Comment by onetruepurple** [comment]
Resolved in v0.16?

**Comment by TerryCavanagh** [comment]
Yep! Thanks, closing!
